emu_sv/ustom_callback_implementations.py

```diff
@@ -0,0 +1,22 @@
+import math
+from typing import Any
+
+import torch
+
+from emu_base.base_classes.config import BackendConfig
+from emu_base.base_classes.default_callbacks import QubitDensity
+from emu_base.base_classes.operator import Operator
+
+from emu_sv import StateVector
+
+
+def custom_qubit_density(
```

Comment:
We should also have a unit test for this function. You could create the Hadamard or bell state or something like that using the string format, for which the qubit density is known, and then run this function and assert that the expected answer comes out.